kubetest/gke.go

```diff
@@ -437,6 +440,7 @@ export KUBE_NODE_OS_DISTRIBUTION='%[3]s'
 		filter := fmt.Sprintf("(metadata.created-by ~ %s)", ig.path)
 		perZoneFilters[ig.zone] = append(perZoneFilters[ig.zone], filter)
 	}
+	isMultizonalOrRegional := len(g.instanceGroups) > 1
```

Comment:
What if we have two instance groups in the same zone?

Reply:
Whoops... that was supposed to be:
```
isMultizonalOrRegional := len(perZoneFilters) > 1
```
Good catch!

Reply:
Done.